lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -48,6 +48,11 @@
     <p class="message-copy">
       ${Text(_("Your {cert_name_long} is being held pending confirmation that the issuance of your {cert_name_short} is in compliance with strict U.S. embargoes on Iran, Cuba, Syria and Sudan. If you think our system has mistakenly identified you as being connected with one of those countries, please let us know by contacting {email}.")).format(email=HTML('<a class="contact-link" href="mailto:{email}">{email}</a>.').format(email=settings.CONTACT_EMAIL), cert_name_short=cert_name_short, cert_name_long=cert_name_long)}
     </p>
+    % elif cert_status['status'] == 'unverified':
+    <p class="message-copy">
+      ${Text(_("Your certificate was not issued because you do not have a current verified identity with edX. "))}
```

Comment:
Replace "edX" with the platform name. edX is our brand, and we try to avoid including it directly in templates to make it easy for others to replace.